js/ui/load_panel.js

```diff
@@ -54,28 +54,9 @@ const LoadPanel = Overlay.inherit({
 
             hideTopOverlayHandler: null,
 
-            /**
-            * @name dxLoadPanelOptions.resizeEnabled
-            * @hidden
-            */
-            resizeEnabled: false,
-
-            focusStateEnabled: false
-
-            /**
-            * @name dxLoadPanelOptions.dragEnabled
-            * @hidden
-            */
```

Comment:
Why did u remove this default values and doccoments? This changes seem to be excess

Reply:
I have compared with upstream 21_2 and those were not there. So I thought it was outdated.

Reply:
Should I add them back?

Reply:
https://github.com/DevExpress/DevExtreme/blob/21_2/js/ui/load_panel.js#L66 it exsist in 21_2